app/gui2/stories/Widgets.story.vue

```diff
@@ -53,8 +52,6 @@ const values = ref(['address', 'age', 'id', 'language', 'location', 'workplace']
       </template>
     </Variant>
     <Variant title="text" :meta="{ customBackground: backgroundColor }">
-      <EnsoTextInputWidget v-model="text" />
-
```

Comment:
What this change is about?

Reply:
Removed a component that no longer exists from stories. Just something I noticed when debugging.